### ISSUE TITLE
Metagene OOM error

### ISSUE THREAD

**Issue by j-berg** [XPRESSyourself/XPRESSpipe#37]
**Describe the bug**
Looks like memory load is too high with current throttling for `metagene`

**To Reproduce**
1. Run metagene with high file load

**Screenshots**
```
Evaluating the metagene profile for...
Processing of batch 1 of 2 complete...
Processing of batch 2 of 2 complete...

Converting positions to metapositions...
Resetting parallelization specs based on max file size to be processed:
Max number of workers: 20
Number of threads per worker (where available): 1
slurmstepd: error: _is_a_lwp: open() /proc/#####/status failed: No such file or directory
slurmstepd: error: Detected 7 oom-kill event(s) in step #######.batch cgroup. Some of your processes may have been killed by the cgroup out-of-memory handler.
(END)

```

**Desktop (please complete the following information):**
HPC

**Comment by j-berg** [comment]
Fixed in 9df8868f0077048e11498823efdd0fab8b189815 and a3e6e232e5044dfa908d804e8c5346952cd389e3

**Comment by j-berg** [comment]
Tests passing, updated in [`XPRESSpipe v0.4.2`](https://github.com/XPRESSyourself/XPRESSpipe/releases/tag/XPRESSpipe-v0.4.2)

**Comment by j-berg** [comment]
Modified BAM file thresholding for parallelization. Fixed by 27eaaed and updated in [`v0.4.3`](https://github.com/XPRESSyourself/XPRESSpipe/releases/tag/XPRESSpipe-v0.4.3)
